lib/radioactive_bunnies/worker.rb

```diff
@@ -1,17 +1,14 @@
 require 'atomic'
 require 'radioactive_bunnies/context'
-
+require 'radioactive_bunnies/deadletter_worker'
 module RadioactiveBunnies::Worker
 
-  def ack!
```

Comment:
I think was designed as a consistent helper function to return true.  The idea is our workers should have used this in the work block.  However, I don't think we did that.  Before I kill it, do we want to put that back and try to enforce that workers should call ack! at the end of the work function.

I can see an advantage here if we later wanted to hook the ack! method to modify successful clean up needed across all workers or even a group of workers?